app/views/components/docs/radio.yml

```diff
@@ -0,0 +1,117 @@
+name: Radio button
+description: A radio button is a GOV.UK element that allows users to answer a question by selecting an option. If you have a question with more than one option you should stack radio buttons.
+body: |
+  Forked from the upcoming [GOV.UK Frontend](https://github.com/alphagov/govuk-frontend), when GOV.UK Frontend release we can replace these source files.
+
+  This should always be used nested in a fieldset so that it has proper context, useful for users of assistive technologies.
+
+  You can also use 'or' as an item to [break up radios](https://www.gov.uk/service-manual/design/writing-for-user-interfaces#ask-questions-that-users-can-understand)
+accessibility_criteria: |
+  Radio buttons should
+
+  - accept focus
+  - be focusable with a keyboard
+  - be usable with a keyboard
+  - indicate when they have focus
+  - change in appearance when touched (in the touch-down state)
+  - be usable with touch
+  - have label with a touch area similar to the input
+  - be usable with [voice commands](https://www.w3.org/WAI/perspectives/voice.html)
+  - have visible text
+  - have a meaningful accessible name
+  - be usable when interacting with the label
+  - additional information in hint text should be read out found when focusing inputs, ideally using `aria-describedby` property.
+
+  [Keyboard interaction](https://www.w3.org/TR/wai-aria-practices-1.1/#radiobutton)
+
+  - when a radio group receives focus:
+    - if a radio button is checked, focus is set on the checked button.
+    - if none of the radio buttons are checked, focus is set on the first radio button in the group.
+  - Space: checks the focused radio button if it is not already checked.
+  - Right Arrow and Down Arrow: move focus to the next radio button in the group, uncheck the previously focused button, and check the newly focused button. If focus is on the last button, focus moves to the first button.
+  - Left Arrow and Up Arrow: move focus to the previous radio button in the group, uncheck the previously focused button, and check the newly focused button. If focus is on the first button, focus moves to the last button.
+accessibility_excluded_rules:
+- radiogroup # Since this is in isolation we don't want to wrap a fieldset here.
+examples:
+  default:
+    data:
+      name: "radio-group"
+      items:
+      - value: "government-gateway"
+        text: "Use Government Gateway"
+      - value: "govuk-verify"
+        text: "Use GOV.UK Verify"
+  with_bold:
+    description: "Used to provide better contrast between long labels and hint text"
```

Comment:
Should this example then include hint text to demonstrate this?

Reply:
I was trying to only use the individual features, but it seems a good idea to show both here good point.